src/helpers/tokens/generate.token.js

```diff
@@ -0,0 +1,23 @@
+import jwt from 'jsonwebtoken';
+import models from '../../api/models';
+import environnements from '../../configs/environnements';
+
+const { Token } = models;
+
+const createToken = (user) => {
+  const token = jwt.sign(
+    { user },
+    environnements.currentEnv.secret,
+    {
+      expiresIn: '6h'
```

Comment:
@alainburindi any reason why you've chosen 6h? Why not 24h? Just want to know your thoughts on this.

Reply:
I thought that 24h is long

Reply:
Okay, please confirm this from the PO. Having users sign in every 6 hours seems inconvenient to me,  but maybe Aaron has a preference we can adopt.